Makefile

```
@@ -100,14 +100,12 @@ update_clang_headers:
 update_fixtures: update_fixtures_macos update_fixtures_docker
 
 update_fixtures_macos:
-	for identifier in org.swift.50120190418a org.swift.5120190905a ; do \
-		swift package reset ; \
-		OVERWRITE_FIXTURES=1 xcrun --toolchain $$identifier swift test ; \
+	for xcode in /Applications/Xcode-11.4.1.app /Applications/Xcode-11.7.app /Applications/Xcode-12.app /Applications/Xcode-12.2.app ; do \
```

Wouldn't it be better to align Xcode's naming conventions with GitHub Actions and Azure Pipelines?
https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#xcode

---

Sure, happy to do that, but this is mostly for maintainers to run locally, not CI. This is how my Xcodes are named so I went with this. As far as I know, you can’t specify Xcode Swift toolchains with their identifiers.

---

Hmm🤔 
How about this one?
```suggestion
	shopt -s nullglob; for xcode in /Applications/Xcode*{11.4.1,11.7,12,12.2}.app ; do \
```

---

Great suggestion, thanks!